migrations/20210401161450_add_notifications_table.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('notifications', table => {
+      table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'));
+      table.string('to').notNull();
```

Comment:
might want an index on this.

Reply:
Do we always have profile_id? I'd be tempted to stick a column for that in as well.